iec61850serversimulator/src/main/java/com/cgi/iec61850serversimulator/dataclass/RelaySchedulePair.java

```diff
@@ -0,0 +1,58 @@
+package com.cgi.iec61850serversimulator.dataclass;
+
+public class RelaySchedulePair {
+    /**
+     * Class to store relay numbers and schedule numbers into a single object for
+     * EventDataListener HashMap and general refactoring of code
+     */
```

Comment:
This JavaDoc should be right above the "public class" line. Generate the JavaDoc for your application locally and check what comes out. The JavaDoc here is probably added as explanation for the relayNr field.

Reply:
Done